### ISSUE TITLE
Remove the need to copy protogen files from lib/teleterm to packages/teleterm

### ISSUE THREAD

**Issue by ravicious** [gravitational/teleport#16621]
Blocked by #15164.

The readme in packages/teleterm explains [how to copy over the protogen files](https://github.com/gravitational/webapps/tree/f0abaf5b9e5a94f6f4212262dd3dbe6167423330/packages/teleterm#generating-tshd-grpc-protobuf-files). Once webapps is merged into teleport (#15164), we should be able to tell `buf` to put the Go protogen files in `lib/teleterm/api/protogen/golang` and the JS files in `webapps/packages/teleterm/src/services/tshd/v1/`.

This is different from #15955 which is about protos for the gRPC server running in the shared process of the Electron app.

**Comment by ravicious** [comment]
Done in gravitational/teleport#20810.
